src/main/kotlin/com.bugsnag.android.gradle/BugsnagUploadProguardTask.kt

```diff
@@ -28,7 +27,7 @@ import java.nio.file.Paths
  * it is usually safe to have this be the absolute last task executed during
  * a build.
  */
-open class BugsnagUploadProguardTask : DefaultTask() {
+abstract class BugsnagUploadProguardTask : DefaultTask(), AndroidManifestInfoReceiver {
```

Comment:
Declaring these tasks as abstract seems to fail the build on AGP 3.4.0/Gradle 5.1. Is it necessary to declare the tasks as abstract in this way or is there a workaround that would support backwards compatibility for these versions?

>GradleException: Cannot create a proxy class for abstract class 'BugsnagManifestUuidTask'.

Reply:
abstract is the standard way to make gradle tasks actually, I'm surprised to see this. I suspect it's an issue with the version of gradle, a cursory check online seems to suggest that this is an issue with using too old of a jdk. Will dig more

Reply:
In talking with @autonomousapps, think this might be just that Gradle 5.1 doesn't support managed properties like this yet. Trying without here: d8cdeb3

Reply:
That seems to have done the trick!